src/pages/survey/[surveyId]/index.tsx

```diff
@@ -30,40 +31,47 @@ function AnswerPage() {
       </Head>
       <Loader isLoading={isLoading} />
       {!isLoading && (
-        <div className="container m-auto mb-6 px-4 text-center md:px-8">
-          <Header>{question}</Header>
+        <>
+          {isSurveyActive ?
+            <div className="container m-auto mb-6 px-4 text-center md:px-8">
+              <Header>{question}</Header>
 
-          <div className="mx-auto  grid  max-w-[500px] grid-cols-2 gap-2 sm:grid-cols-4">
-            {icons.map((icon, idx) => (
-              <EmojiButton
-                icon={icon}
-                selected={selectedIcon === icon}
-                key={idx}
-                onClick={handleIconClick}
-              />
-            ))}
-          </div>
-          <div className="mt-8">
-            <textarea
-              className="h-56 w-[500px] max-w-[100%] resize-none rounded-lg p-4 shadow focus:outline-none"
-              placeholder="Tell Us More"
-              value={answer}
-              onChange={handleInputAnswer}
-            ></textarea>
-          </div>
-          <div className="flex justify-center">
-            <Button
-              disabled={!selectedIcon || buttonDisable}
-              onClick={handleSave}
-              className="mt-6 w-full sm:w-auto"
-              variant={ButtonVariant.PRIMARY}
-              sizeType={ButtonSize.MEDIUM}
-              isLoading={isAnswering}
-            >
-              Send
-            </Button>
-          </div>
-        </div>
+              <div className="mx-auto  grid  max-w-[500px] grid-cols-2 gap-2 sm:grid-cols-4">
+                {icons.map((icon, idx) => (
+                  <EmojiButton
+                    icon={icon}
+                    selected={selectedIcon === icon}
+                    key={idx}
+                    onClick={handleIconClick}
+                  />
+                ))}
+              </div>
+              <div className="mt-8">
+                <textarea
+                  className="h-56 w-[500px] max-w-[100%] resize-none rounded-lg p-4 shadow focus:outline-none"
+                  placeholder="Tell Us More"
+                  value={answer}
+                  onChange={handleInputAnswer}
+                ></textarea>
+              </div>
+              <div className="flex justify-center">
+                <Button
+                  disabled={!selectedIcon || buttonDisable}
+                  onClick={handleSave}
+                  className="mt-6 w-full sm:w-auto"
+                  variant={ButtonVariant.PRIMARY}
+                  sizeType={ButtonSize.MEDIUM}
+                  isLoading={isAnswering}
+                >
+                  Send
+                </Button>
+              </div>
+            </div> : <div className='container m-auto mb-6 px-4 text-center md:px-8'>
+              <h1 className='text-5xl'>🙁</h1>
+              <h1 className='mt-5 text-xl'>Oops,, Survey is no longer active</h1>
+            </div>
```

Comment:
Maybe let's add also button that will redirect user to home page :)

Reply:
Sure :)